src/components/app/app.scss

```diff
@@ -42,6 +63,27 @@
   --color-run-list-hover: #{colors.$slate-300};
   --color-base-20: #{colors.$grey-800};
   --color-black-10: #{colors.$slate-700};
+
+  // Experiment tracking colors below
+  --color-exp-tracking-bg: #{colors.$slate-900};
+  --color-exp-tracking-metadata: #{colors.$slate-900};
+  --color-exp-tracking-datasets: #{colors.$ocean-900};
+  --color-metrics-plot-text: #{colors.$grey-900};
+  --color-metrics-plot-text-bold: #{colors.$white-0};
```

Comment:
@tynandebold  -- these colours are specific to parellel coordinates, should we rename them to add parellel-coordinates in the name like we did for timeseries

Reply:
We should see if there are any duplicates first before renaming. Checking now, I see that we use the color `$grey-900` for both `--color-metrics-plot-text` and for `--color-metrics-plot-time-series-axis` for the dark theme. We should be able to share that color somehow and not have two variables for it. What do you think?

Reply:
but they don't match on light theme. one is grey 200 and one is grey 300